cypresse2e/cypress/integration/e2eTests/aboutUs/aboutUs.spec.js

```diff
@@ -0,0 +1,77 @@
+import AboutUs from '../../../elements/pages/aboutUs'
+
+
+describe('Fight Pandemics About Us page', () => {
```

Comment:
'Fight Pandemics' should be 'FightPandemics'

Reply:
Done!